lke_clusters_acl.go

```diff
@@ -0,0 +1,68 @@
+package linodego
+
+import "context"
+
+// LKEClusterControlPlaneACLAddresses describes the
+// allowed IP ranges for an LKE cluster's control plane.
+type LKEClusterControlPlaneACLAddresses struct {
+	IPv4 []string `json:"ipv4,omitempty"`
+	IPv6 []string `json:"ipv6,omitempty"`
+}
+
+// LKEClusterControlPlaneACL describes the ACL configuration
+// for an LKE cluster's control plane.
+type LKEClusterControlPlaneACL struct {
+	Enabled   *bool                               `json:"enabled,omitempty"`
```

Comment:
Making `Enabled` a `*bool` feels a bit wrong but is necessary to make sure users can disable ACL using the PUT endpoint.